static/scss/navbar.scss

```diff
@@ -285,3 +282,7 @@ $header_font_color: white;
   text-decoration: none;
   font-weight: 300;
 }
+
+.mdl-layout-title > a:hover, .mdl-layout-title > a:focus {
+  color: white;
+}
```

Comment:
This keeps the "MITx Micromasters" link in the navbar from changing to the default hover color.